lesson01/assignment/scripts.js

```diff
@@ -30,4 +34,96 @@ document. addEventListener("DOMContentLoaded", function(){
     inputField.value = ''
   })
 
-})
\ No newline at end of file
+  form1.addEventListener('submit', function(event){
+    // Prevent the form from trying to submit to a server
+    event.preventDefault()
+
+    // An array that we can transform based on user input
+    let data1 = ['apples', 'bananas', 'carrots', 'bread']
+
+    const inputField1 = document.querySelector('#form1 .input')
+    const userItem = inputField1.value
+
+
+    // If the user's input was a word, push it into the array and continue
+    if(isNaN(userItem)){
+      data1.push(userItem)
+
+      // Output the total to the page- no need to convert the numeric sum back to a string, HTML will just print the number as a string
+      document.querySelector('#form1 .output').innerHTML = data1
+    }
+
+    var eggs = data1.includes('eggs');
+    document.getElementById('eggs').innerHTML = eggs;
+
+    // Clear the form field so the user can try again
+    inputField1.value = ''
+  })
+
+  form2.addEventListener('submit', function(event){
+    // Prevent the form from trying to submit to a server
+    event.preventDefault()
+
+    // An array that we can transform based on user input
+    let numlist = [2 , 3 , 4 , 5 , 6]
+
+    const inputField2 = document.querySelector('#form2 .input')
+    const userItem2 = inputField2.value
+    const userNumber2 = parseInt(userItem2)
+
+    // If the user's input was a number, push it into the array and continue
+    if(!isNaN(userNumber2)){
+      let x = numlist.map(numlist => numlist * userNumber2);
+      document.querySelector('#form2Num').innerHTML = userNumber2;
+      document.querySelector('#form2 .output').innerHTML = x;
+    }
```

Comment:
Nice user experience here, printing both their input and the result. You could consider printing everything in one statement. If the `p` tag holding output had its own selector, code might look like: 
```
output.innerHTML = `List of Numbers multiplied by ${userNumber2} is ${x}`
```